cmd/collectors/zapiperf/README.md

```diff
@@ -119,7 +129,7 @@ This section defines the list of counters that will be collected. These counters
 
 Some counters require a "base-counter" for post-processing. If the base-counter is missing, ZapiPerf will still run, but the missing data won't be exported.
 
-The display name of a counter can be changed with `=>` (e.g. `nfsv3_ops => ops`). The special counter `instance_name` will be renamed to the value of `object` by default.
+The display name of a counter can be changed with `=>` (e.g., `nfsv3_ops => ops`). The special counter `instance_name` will be renamed to the value of `object` by default.
```

Comment:
There's one conversion Harvest does for you by default, the `instance_name` counter will be renamed to the value of `object`.